airbyte-integrations/connectors/source-mailchimp/Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM airbyte/integration-base-python:dev
+FROM python:3.7.9-alpine3.12
 
 # Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
+#RUN apt-get update && apt-get install -y bash && rm -rf /var/lib/apt/lists/*
 
 ENV CODE_PATH="source_mailchimp"
 ENV AIRBYTE_IMPL_MODULE="source_mailchimp"
 ENV AIRBYTE_IMPL_PATH="SourceMailchimp"
 
 WORKDIR /airbyte/integration_code
 COPY $CODE_PATH ./$CODE_PATH
+COPY /build/wheels/* ./build/wheels/
 COPY setup.py ./
+RUN pip install build/wheels/airbyte_protocol-0.0.0-py3-none-any.whl
```

Comment:
This feels kind of arbitrary to ask a user to include (as opposed to assuming that once we copy the `build` directory, `setup install .[main]` is sufficient). Is there a way to make it so that this can all be taken care of inside `pip install .` ?